python/plugins/processing/algs/saga/SagaAlgorithm.py

```diff
@@ -396,12 +396,6 @@ def exportRasterLayer(self, parameterName, layer):
         else:
             filename = os.path.basename(layer.source())
 
-        validChars = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789:'
```

Comment:
I think the Travis issue is stemming from allowing "." characters now

Reply:
Good catch. I added that code back, just in case